cfgmgr/teammgr.cpp

```diff
@@ -447,12 +447,26 @@ task_process_status TeamMgr::addLagMember(const string &lag, const string &membe
 
     stringstream cmd;
     string res;
+    string PoId, PoId2;
+    size_t found = lag.find('l');
+    if (found == string::npos)
+    {
+        SWSS_LOG_ERROR("Failed to parse %s", lag.c_str());
+        return task_failed;
+    }
+    PoId = lag.substr(found+1);
+    /* Ignore all 0's in the Port-channel id*/
+    PoId2 = PoId.erase(0, min(PoId.find_first_not_of('0'), PoId.size()-1));
```

Comment:
this part of logic can be improved, it should try to find the first digit, then use stol to get an integert and use it later.